manuscript/the-meat-start.txt

```diff
@@ -38,17 +38,15 @@ Which compiles to:
         document.querySelectorAll(‘.h1bgraph’)[0]
     );
 
-As you can see, HTML code translates to `React.createElement` calls with attributes translated into a property dictionary. The beauty of this approach is that you can use React components as if they were HTML tags and that HTML attributes can be anything.
+As you can see, HTML code translates to `React.createElement` calls with attributes translated into a property dictionary. The beauty of this approach is two-pronged: you can use React components as if they were HTML tags and  HTML attributes can be anything.
 
 You’ll see that anything from a simple value to a function or an object works equally well.
 
-I'm not sure yet whether this is better than separate template files in Mustache or whatever. There are benefits to both approaches. I mean, would you let a designer write the HTML inside your JavaScript files? I wouldn't.
+I'm not sure yet whether this is better than separate template files in Mustache or whatever. There are benefits to both approaches. I mean, would you let a designer write the HTML inside your JavaScript files? I wouldn't, but it's definitely better than manually `+`-ing strings or Angular's approach of putting everything into HTML. Considerably better.
 
-But it's definitely better than manually `+`-ing strings or Angular's approach of putting everything into HTML. Much better.
+If you skipped the setup section and don't have a JSX compilation system set up, you should do that now. You can also use the project stub you got with the book.
 
-If you skipped the setup section and don't have a JSX compilation system set up, I suggest you do that now. You can also use the project stub you got with the book.
-
-Just like the official documentation, I don't suggest using `JSXTransform.js`. Sure it's neat to have all this compilation happen right in the browser, but it's bad practice.
+Just like the official documentation, I don't suggest using `JSXTransform.js`. Sure, it's neat to have all this compilation happen right in the browser, but it's bad practice.
```

Comment:
We should remove this whole paragraph. It isn't all that relevant, doesn't add much, and is potentially confusing.